docs/checks/streetNameCheck.md

```diff
@@ -0,0 +1,46 @@
+# StreetNameCheck
+
+#### Description
+This check flags all the objects (Nodes, Ways and Relations) with "ss" or "ß" found in objects. In addition, it flags 
+Relations with deprecated tags as seen on  [OSMOSE](https://osmose.openstreetmap.fr/en/errors?item=9009).
```

Comment:
I understand that this is all part of the same osmose check, but it seems a bit odd to flag deprecated tags in the same check as flagging name/address_street tag issues.